FirebaseMLModelDownloader/Sources/ModelDownloadTask.swift

```diff
@@ -89,14 +89,34 @@ extension ModelDownloadTask: URLSessionDownloadDelegate {
     return "fbml_model__\(appName)__\(remoteModelInfo.name)"
   }
 
+  /// Handle client-side errors.
   func urlSession(_ session: URLSession, task: URLSessionTask, didCompleteWithError error: Error?) {
-    // TODO: Handle model download url expiry and other download errors
+    assert(task == downloadTask)
```

Comment:
We shouldn't assert here, if anything just log an internal error and return. Never want to crash a customer's app.

Reply:
Gotcha! Added a TODO for logging.